frontend-plugin-core/src/main/java/com/github/eirslett/maven/plugins/frontend/lib/PnpmInstaller.java

```diff
@@ -169,42 +173,72 @@ private void installPnpm() throws InstallationException {
         }
     }
 
-    private void copyPnpmScripts() throws InstallationException{
-        File installDirectory = getNodeInstallDirectory();
-
-        File nodeModulesDirectory = new File(installDirectory, "node_modules");
-        File pnpmDirectory = new File(nodeModulesDirectory, "pnpm");
-        // create a copy of the pnpm scripts next to the node executable
-        for (String script : Arrays.asList("pnpm", "pnpm.cmd")) {
-            File scriptFile = new File(pnpmDirectory, "bin" + File.separator + script);
-            if (scriptFile.exists()) {
-                File copy = new File(installDirectory, script);
-                if (!copy.exists()) {
-                    try
-                    {
-                        FileUtils.copyFile(scriptFile, copy);
-                    }
-                    catch (IOException e)
-                    {
-                        throw new InstallationException("Could not copy pnpm", e);
-                    }
-                    copy.setExecutable(true);
-                }
-            }
```

Comment:
I have removed this old code as it seems to have been copied from the `NpmInstaller`, but never actually functioned. I've manually checked every release of PNPM, and none of them include a `pnpm` or `pnpm.cmd` file in their `bin` directory.